packages/react-components/src/templates/__tests__/SharedResearchOutput.test.tsx

```diff
@@ -80,10 +80,12 @@ it('displays access instructions when data provided', () => {
 
 it('displays contact pm card when there are contact emails', () => {
```

Comment:
Test name

Reply:
Leaving this as is, as we are asserting on the word "contact pm"